package/Dockerfile

```
@@ -38,9 +38,9 @@ ENV HELM_VERSION v3.3.1
 ENV KUSTOMIZE_VERSION v3.5.4
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=0.3.900
+ENV CATTLE_FLEET_MIN_VERSION=0.3.1000-security1+up0.3.9
```

We need to update the version following https://github.com/rancher/charts/pull/2345:
```suggestion
ENV CATTLE_FLEET_MIN_VERSION=0.3.1000-security1+up0.3.10
```